lib/cli/commands/addcurrency.ts

```diff
@@ -0,0 +1,38 @@
+import { Arguments } from 'yargs';
+import { callback, loadXudClient } from '../command';
+import { AddCurrencyRequest } from '../../proto/xudrpc_pb';
+import { SwapClients } from '../../types/enums';
+
+export const command = 'addcurrency <currency> <swap_client> [decimal_places] [token_address]';
+
+export const describe = 'add a supported currency';
+
+export const builder = {
+  currency: {
+    description: 'The ticker symbol for the currency',
+    type: 'string',
+  },
+  swap_client: {
+    description: 'The payment channel network client for swaps',
+    type: 'string',
+    choices: ['LND', 'RAIDEN'],
+  },
+  decimal_places: {
+    description: 'The places to the right of the decimal point of the smallest subunit (e.g. satoshi)',
+    default: 8,
```

Comment:
Idea for the future: automatically detect `decimal_places` of ERC20s if nothing is specified. The users are running an Ethereum node anyways.